algorithmic_efficiency/workloads/librispeech_conformer/librispeech_pytorch/workload.py

```diff
@@ -60,8 +60,10 @@ def init_model_fn(
     model.to(DEVICE)
     if N_GPUS > 1:
       if USE_PYTORCH_DDP:
+        self.requires_sync_before_eval = True
         model = DDP(model, device_ids=[RANK], output_device=RANK)
       else:
+        self.requires_sync_before_eval = False
```

Comment:
I think you should add this False line to before `if N_GPUS > 1:`

Reply:
Yes! Thank you so much for catching this!!!